### PR TITLE
feat(textarea): resize prop

### PR DESCRIPTION
**Pull request by PaulPCIO**
## Checklist

- [ ] Fix linting errors
- [ ] Tests have been added / updated (or snapshots)

## Change information

Included prop for the textarea which allows setting the resize css value.

### CODE REVIEW

**Comment by PaulPCIO** [comment]
resolves https://github.com/geist-org/react/issues/415

File: components/textarea/textarea.tsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import withDefaults from '../utils/with-defaults'
 import { NormalTypes } from '../utils/prop-types'
 import { getColors } from '../input/styles'
 
+type ResizeTypes = 'none' | 'both' | 'horizontal' | 'vertical' | 'initial' | 'inherit'
```
Comment (unix) [inline components/textarea/textarea.tsx:7]: `ResizeTypes` should be a `tuple`:
```
const resizeTypes = tuple('none', ...)
type ResizeTypes = typeof resizeTypes[number]
```

File: components/textarea/textarea.tsx [hunk 2]
```diff
@@ -17,6 +19,7 @@ interface Props {
   onFocus?: (e: React.FocusEvent<HTMLTextAreaElement>) => void
   onBlur?: (e: React.FocusEvent<HTMLTextAreaElement>) => void
   className?: string
+  resize?:ResizeTypes
```
Comment (unix) [inline components/textarea/textarea.tsx:22]: There should be a space where you can run prettier to verify the code style.

File: components/textarea/textarea.tsx [hunk 3]
```diff
@@ -27,6 +30,7 @@ const defaultProps = {
   disabled: false,
   readOnly: false,
   className: '',
+  resize:'initial' as ResizeTypes
```
Comment (unix) [inline components/textarea/textarea.tsx:33]: I looked at the documentation and the initial value is `none`?

File: pages/en-us/components/textarea.mdx [hunk 4]
```diff
@@ -135,6 +135,7 @@ Retrieve multi-line user input.
 | **width**        | set width string      | `string`                         | -                                                       | `initial` |
 | **minHeight**    | set min-height string | `string`                         | -                                                       | -         |
 | **onChange**     | change event          | `(e: React.ChangeEvent) => void` | -                                                       | -         |
+| **resize**       | change event          | `string`                         | `'none', 'both', 'horizontal', 'vertical', 'initial', 'inherit'`   | `initial` |
```
Comment (unix) [inline pages/en-us/components/textarea.mdx:138]: The content of the description needs to be optimized.

**Review by unix — CHANGES_REQUESTED** [review]
Thank you for your contribution 👍 , there are a few minor issues here that need to be followed up.

File: components/textarea/textarea.tsx (hunk 1 above)
Comment (PaulPCIO) [reply components/textarea/textarea.tsx]: updated commit: https://github.com/geist-org/react/pull/416/commits/ec1ba50fb476e625f49f631a7c41426db247a8e0

**Pull request merged**